### DISCUSSION TITLE
application for GitHub Education benefits

### DISCUSSION THREAD

**Discussion by sonu-verma108** [community/community#21122] [GitHub Education]
Hi my name is SONU VERMA. I have submitted the application for GitHub Education benefits but there is no response till now. please help me
.........................

**Comment by nouhelion** [comment]
_Usually they take a month or just a few weeks (that's how much time it took for me ) , did you use a your school/university email ? if so they shouldn't take that long , also it might help if u provide them with your university card it will be a decent proof .
i hope my answer helps in any way ._
